test/decompile/basic.txt

```diff
@@ -80,9 +80,9 @@ import function ns_fi();
 
 export function f(a:int, b:int):int {
   var c:long = 8L;
-  var d:float = 6f;
-  var e:double = 7d;
-  if (e < 10d) { 1[4]:int = (2[3]:int + 5) }
+  var d:float = 6.0f;
+  var e:double = 7.0;
+  if (e < 10.0) { 1[4]:int = (2[3]:int + 5) }
```

Comment:
Which part caused this to change?

Reply:
It uses ostringstream before, which formats `1.0` as `1`. It now uses `std::to_string` which makes it into `1.000000`, which I decided to turn into `1.0` :)